src/ui/modules/blog/PostContent.tsx

```
@@ -0,0 +1,60 @@
+import moduleProps from '@/lib/moduleProps';
+import { cn } from '@/lib/utils';
+import Date from '@/ui/Date';
```

Biome (1.9.4)

[error] Do not shadow the global "Date" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)